src/components/tasklist/Tasklist.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import './tasklist.scss';
+
+import { FaTrashAlt, FaBatteryThreeQuarters, FaCheck } from 'react-icons/fa';
+
+class Tasklist extends Component {
+
+    userId = 3;
+
+    noResults = "No Results";
+
+    state = {
+        todos: [],
+        displayFilter: []
+    };
+
+    componentDidMount() {
+        fetch(`https://jsonplaceholder.typicode.com/todos?userId=${ this.userId }`)
+            .then(response => response.json())
+            .then(todos => {
+                return this.setState({todos})
+            })
+    }
+
+    deleteTask = () => {
+        console.log('deleted')
+    }
+    inProgress = () => {
+        console.log('in progress')
+    }
+    completeTask = (key, e) => {
+        const userTasks = this.state.todos.map(
+            el => {
+                if (el.completed === false) {
+                    el.id === key ? el.completed = true : el.completed = el.completed;
```

Comment:
`el.completed = el.completed` why do you assign value to itself? It is useless expression.